src/Catalog/Registration/RegistrationMakerCatalogItem.cs

```diff
@@ -120,7 +122,7 @@ Uri GetPackageContentAddress()
                 {
                     string id = _catalogItem.GetTriplesWithSubjectPredicate(subject, _catalogItem.CreateUriNode(Schema.Predicates.Id)).FirstOrDefault().Object.ToString().ToLowerInvariant();
                     string version = _catalogItem.GetTriplesWithSubjectPredicate(subject, _catalogItem.CreateUriNode(Schema.Predicates.Version)).FirstOrDefault().Object.ToString().ToLowerInvariant();
-                    string path = string.Format("packages/{0}.{1}.nupkg", id.ToLowerInvariant(), version.ToLowerInvariant());
+                    string path = GetPackagePath(id, version);
```

Comment:
A more readable approach imo is to:

* Either have an interface "IPackagePathProvider" with several implementations that can be injected into "RegistrationMakerCatalogItem"
* Or there's a new class that inherits RegistrationMakerCatlogItem and has an overloaded method

The main issue I have with replacing these shorthands is that it's not obvious from looking at the code what it does. E.g. if I stare at this code but the caller has replaced the function, I have to inspect the function variable first to see what it does. Don't have a big issue with the syntax/readability of the syntax, more with having a quick understanding without having to look into things at runtime.